.github/workflows/buildAndTest.yml

```diff
@@ -75,6 +80,7 @@ jobs:
       uses: ./.github/actions/setup-build
       with:
         cache-suffix: 'build-${{ matrix.llvm-build }}'
```

Comment:
I suggest adding `-${{ matrix.torch-version }}` to the cache suffix, because otherwise, we might start seeing build failures when a workflow tries to create a cache entry that already exists.